tools/ratbagd.py

```diff
@@ -347,6 +359,11 @@ def name(self):
         """The device name, usually provided by the kernel."""
         return self._get_dbus_property("Name")
 
+    @GObject.Property
+    def device_type(self):
+        """The type of this device: unspecified(0), mouse(1), keyboard(2) or other(3)"""
+        return RatbagDeviceType(self._get_dbus_property("DeviceType"))
+
```

Comment:
Description doesn't equal return value.

Reply:
What would be better?

`The type of this device as specified in the .device file` ?

or would be `The type of this device` good enough?

Reply:
`The device type, see RatbagDeviceType.` is good enough IMHO. I think pointing out the enum explicitly is a good idea.